edenia/antelope-tools
# Feat/network name in page titles 1302

### Add the network name in the titles

### What does this PR do?

- Resolve #1302
- Fix missing h1 and skipped heading levels
- Mention the current network in the About page 

### Steps to test

1. Check the changes in the languages files
2. Run the project locally
3. Check the titles

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/4488e744-0380-4616-a6e5-c9f6f1ab3a25)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/f139ba0d-3c2f-4d7d-b079-69eec57e8cb2)
